server/channels/app/helper_test.go

```diff
@@ -111,6 +108,7 @@ func setupTestHelper(dbStore store.Store, enterprise bool, includeCacheLayer boo
 		TestLogger:        testLogger,
 		IncludeCacheLayer: includeCacheLayer,
 		ConfigStore:       configStore,
+		TB:                tb,
```

Comment:
A reviewer pointed out on https://github.com/mattermost/mattermost/pull/30707#discussion_r2073353799 that storing a `testing.TB` in a struct can cause issues with sub-tests. Would you please migrate the code to just pass a testing.TB to all methods that need it. Thanks!